### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by Loltolo-Lesapiti**
In this milestone, I used CSS to improve the look and feel of the application. There, this project entails a styled leaderboard application that;

- Sends and receives data from an API
- uses API documentation and 
- Demonstrates working with JSON files using JavaScript code asynchronous.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,13 +5,17 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>LeaderBoard</title>
+  <link rel="stylesheet" href="style.css">
```
Comment (ShahierNashaat) [inline src/index.html]: - [ ] Kindly remove the link to the CSS file as you are already imported the file into the `index.js` file.

File: src/index.html [hunk 2]
```diff
@@ -26,7 +30,7 @@ <h3>Add Your score</h3>
       </form>
     </div>
   </div>
-  <footer><p>LeaderBoard ©2022 (Petro Lesapiti)</p></footer>
-  <script type="module" src="index.js"></script>
+  <footer><p> © WORLD ATLETICS 2022</p></footer>
+  <script src="index.js"></script>
```
Comment (ShahierNashaat) [inline src/index.html]: - [ ] Kindly remove the link to the JS file as the webpack already knows the entry point file.

File: dist/index.html [hunk 3]
```diff
@@ -5,13 +5,17 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>LeaderBoard</title>
+  <link rel="stylesheet" href="style.css">
```
Comment (ShahierNashaat) [inline dist/index.html]: - [ ] Also here remove the link to the CSS file as you are already imported it into the JS file.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Loltolo-Lesapiti, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Loltolo-Lesapiti , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**